feature/mypage/src/main/java/com/mument_android/mypage/MyPageViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package com.mument_android.mypage
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.mument_android.mypage.data.NoticeData
+import com.mument_android.mypage.data.UserData
+
+class MyPageViewModel : ViewModel() {
+
+    //myPage
+    private val _isBtnClick = MutableLiveData(false)
+    val isBtnClick = _isBtnClick
```

Comment:
이런 방식으로 바꿔주세요 :) 뷰모델 외부에서 해당 방식으로 isBtnClick을 접근한다면 MutableLiveData한 상태의 값을 받을 수 있기 때문에 외부에서 값을 바꿀 수 있습니다. 읽기 전용으로만 접근할 수 있게 해주세요 !!
```suggestion
    private val _isBtnClick = MutableLiveData(false)
    val isBtnClick :LiveData<Boolean>get() =_isBtnClick
```

Reply:
아래 _noticeList도 동일!

Reply:
알겠숩니다!